Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -0,0 +1,2138 @@
+            ro = RequestObject()
+            ro.set_http_method('GET')
+            ro.set_owner(owner)
+            ro.set_request_uri('/v2/indicators/' + str(apiBranch) + "/" + quote(indicator_value).replace("/", "%2F"))
+            results = tc.api_request(ro)
+            if results.headers['content-type'] == 'application/json':
+                if results.json()['status'] == 'Success':
+                    res = results.json()['data'][item['apiEntity']]
+                    res['ownerName'] = res['owner']['name']
+                    res['type'] = item['name']
+                    stdout.append(res)
+                    break
+
+    return stdout
+
+
+def get_indapi(indicator_value, owner, tc):
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_owner(owner)
+    ro.set_request_uri('/v2/indicators?filters=summary/v2/indicators?filters=summary%3D'
+                       + quote(indicator_value).replace("/", "%2F"))
+    results = tc.api_request(ro)
+    if results.headers['content-type'] == 'application/json':
+        return results.json()
+    else:
+        return {}
+
+
+def get_indicatorOwner(indicator_value, owner=demisto.params()['defaultOrg']):
+    tc = get_client()
+    indsowners = {}
+    types = tc_get_indicator_types_request()['data']['indicatorType']
+    for item in types:
+        apiBranch = item['apiBranch']
+        ro = RequestObject()
+        ro.set_http_method('GET')
+        ro.set_owner(owner)
+        ro.set_request_uri('/v2/indicators/{}/{}/owners'.format(apiBranch, quote(indicator_value).replace("/", "%2F")))
+        results = tc.api_request(ro)
+        if results.headers['content-type'] == 'application/json':
+            ownersRaw = results.json()
+            if 'status' in ownersRaw:
+                if ownersRaw['status'] == 'Success':
+                    if len(ownersRaw['data']['owner']) > 0:
+                        indsowners = results.json()
+                        break
+    return indsowners
+
+
+# pylint: disable=E1101
+def get_indicators(indicator_value=None, indicator_type=None, owners=None, rating_threshold=-1, confidence_threshold=-1,
+                   freshness=None, associated_groups=False, associated_indicators=False, include_observations=False,
+                   include_tags=False, loopowners=False):
+    tc = get_client()
+    raw_indicators = []
+    if owners and owners.find(",") > -1:
+        owners = owners.split(",")
+        for owner in owners:
+            raw_indicators = get_xindapi(indicator_value, indicator_type, owner, tc)
+            if len(raw_indicators) > 0:
+                owners = owner
+                break
+    else:
+        raw_indicators = get_xindapi(indicator_value, indicator_type, owners, tc)
+        if len(raw_indicators) == 0 and loopowners is True:
+            owners = get_indicatorOwner(indicator_value)
+            if 'data' in owners:
+                if 'owner' in owners['data']:
```

Comment:
you can join those conditions:
```suggestion
            if 'owner' in owners.get('data', {}):
```